charts/rollups-node/Chart.yaml

```diff
@@ -13,7 +13,7 @@ maintainers:
     url: https://github.com/oap75
 
 type: application
-version: 1.0.1-0
+version: 1.0.1-1
```

Comment:
Since your targeting `next` branch, I'd use this commit as a `DELETE: ` one, just to make CI pass, and we manage the version bump on another individual commit.